### ISSUE TITLE
Allow all connections to be stored in User.Settings

### ISSUE THREAD

**Issue by sftgoldie** [Borvik/vscode-postgres#86]
Far as I can tell, I would have to update each of my dev machines with the connections to use this plugin. Instead is it possible to store all the connections in `settings.json` instead? For example `vscode-sqltools` allows this.

By default, store connections in secure storage, but for unsecure connections (a la dev dbs) allow for plain text settings.

**Comment by kyhou** [comment]
Yeah, I'm currently transferring my settings to a new computer here in my job, and now I have to remake all my connections manually (16 of them)

**Comment by jrussellsmyth** [comment]
This would be particularly helpful for us as we are using devcontainers and each project has a preconfigured container - being able to include the db connection in settings provided by the container puts us one step closer to instant-dev-ready

**Comment by jbjonesjr** [comment]
Echo the value of this old thread. Moving between different systems is hard as these settings are not persisted as a a vscode configuration (unlike other db plugins that do manage it via settings). Not sure if https://github.com/Borvik/vscode-postgres/issues/210 would bring some of these improvements, but figured i'd poke
